CHANGELOG.md

```diff
@@ -139,6 +139,7 @@ You can find its changes [documented below](#070---2021-01-01).
 - X11: window focus events ([#1938] by [@Maan2003]
 - Preserve the aspect ratio of a clipped region in an Image ([#2195] by [@barsae])
 - GTK: Hot state now properly resets when the mouse leaves the window via an occluded part. ([#2324] by [@xStrom])
+- Windows: generate `commands::OPEN_FILES` when using multiselect in file open dialog. ([#2332] by [@ppykalo])
```

Comment:
Unfortunately the links aren't autogenerated. Please manually define `[#2332]` at the bottom of the file.